syntax="proto3";
package weight_raft;
option cc_generic_services = true;

message WeightRequest {
  int64 device_id = 1;
  int64 weight = 2;
};

message WeightResponse {
  bool success = 1;
  int64 data = 2;
  optional string redirect = 3;
};

service WeightService {
  rpc setWeight(WeightRequest) returns (WeightResponse);
  rpc getWeight(WeightRequest) returns (WeightResponse);
};